ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: unrecognized arguments: -n --dist=no
  inifile: None
  rootdir: /home/pranjal/sem6_courses/stt/month2/lab6/algorithms


real	0m0.168s
user	0m0.137s
sys	0m0.032s
